src/adapters/SensorApplication.java

```diff
@@ -0,0 +1,115 @@
+/**
+ * CS 410 - Assignment 2 
+ * @author Daniyal Adzhiyev
+ * @version 1.0 05/28/2019
+ * 
+ * This Application uses the Adapter Software Pattern to create adapter's for multiple predefined sensor classes.
+ * Where the adapter classes implement the Sensor Application Interface, to 'adapt' the original sensor classes
+ * so that all of the adapter classes have the same methods needed to display the information in the Sensor 
+ * Application.
+ *  
+ */
+
+package adapters;
+
+import java.awt.Dimension;
+import java.awt.GridBagConstraints;
+import java.awt.GridBagLayout;
+import java.awt.GridLayout;
+import java.awt.Insets;
+
+import javax.swing.JFrame;
+import javax.swing.JLabel;
+import javax.swing.JPanel;
+import javax.swing.border.TitledBorder;
+
+import sensor.PressureSensor;
+import sensor.RadiationSensor;
+import sensor.TemperatureSensor;
+
+/*
+ * This Class creates the GUI for the Sensor Application
+ */
+public class SensorApplication extends JFrame {
+	public SensorApplication(SensorApplicationInterface[] sensor) {
+		setTitle("Sensor Tracker"); // set's the title of the application
+		setLayout(new GridLayout(3, 1)); // uses a gridlayout
+
+		// Uses grid bag constraints to seperate the Gauge Panels and the Message
+		// Textfields
+		GridBagConstraints c = new GridBagConstraints();
+		c.insets = new Insets(10, 10, 10, 10);
+		c.gridx = 0;
+		c.gridy = 1;
+
+		// Creates the first Sensor Panel
+		JPanel firstSensorPnl = new JPanel();
+		firstSensorPnl.setLayout(new GridBagLayout());
+		// sets the panel title to the first sensor name
+		firstSensorPnl.setBorder(new TitledBorder(sensor[0].getSensorName()));
+		add(firstSensorPnl); // adds panel to the JFrame
+		JPanel firstSensorGaugePnl = new JPanel(); // JPanel to represent the Sensor Gauge
+		// sets the size of the JPanel Gauge based on length returned form the sensor
+		firstSensorGaugePnl.setPreferredSize(new Dimension(sensor[0].getLength(), 100));
+		// sets the background color of the JPanel Gauge based on the sensor color code
+		firstSensorGaugePnl.setBackground(sensor[0].getColorCode());
+		// creates JLabel and sets the text to the sensor's getMessage() method
+		JLabel firstSensorMessageLbl = new JLabel(sensor[0].getMessage());
+
+		firstSensorPnl.add(firstSensorGaugePnl); // adds gauge to panel
+		firstSensorPnl.add(firstSensorMessageLbl, c); // adds message label to panel, with constraints
+
+		// Creates the second Sensor Panel
+		JPanel secondSensorPnl = new JPanel();
+		secondSensorPnl.setLayout(new GridBagLayout());
+		// sets the panel title to the second sensor name
```

Comment:
I would recommend removing some of these comments to improve readability. Much of the code is self explanatory, such as JPanel secondSensorPnl=new JPanel() is clearly creating a new JPanel and the name tells you it is the second sensor panel.